genie-agent-app/build.gradle

```diff
@@ -25,6 +25,14 @@ license {
     exclude "*.txt"
 }
 
+dependencyManagement {
+    imports {
+        mavenBom "com.amazonaws:aws-java-sdk-bom:1.12.99"
```

Comment:
`com.amazonaws` is AWS SDK V1 and with Amazon's [announcement of end-of-support for AWS SDK for Java v1.x](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-aws-sdk-for-java-v1-x-on-december-31-2025/) you'll have to move to V2 eventually
(jFYI, both V1 and V2 will continue being supported with upgrade to Spring Boot for now)